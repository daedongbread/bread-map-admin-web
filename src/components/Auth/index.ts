export * from './PersistLogin';
export * from './RequireAuth';
export * from './ControlInterceptor';
