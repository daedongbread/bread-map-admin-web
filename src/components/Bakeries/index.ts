export * from './BakeriesTable';
export * from './BakeryFilter';
export * from './BakeryTotalCount';
export * from './BakeryInfoAndFilter';
