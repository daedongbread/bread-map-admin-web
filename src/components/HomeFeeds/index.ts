export * from './HomeFeedsTable';
export * from './FeedSearchBar';
