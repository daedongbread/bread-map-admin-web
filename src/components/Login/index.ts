export * from './Logo';
export * from './LoginForm';
