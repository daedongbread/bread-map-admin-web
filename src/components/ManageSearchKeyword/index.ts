export * from './ManageSearchKeywordCountTable';
export * from './ManageSearchKeywordRankTable';
