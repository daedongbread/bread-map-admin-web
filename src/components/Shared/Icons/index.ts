export * from './Search';
export * from './Pencil';
export * from './BarGraph';
export * from './ChevronDown';
export * from './ChevronLeft';
export * from './ChevronRight';
export * from './Frown';
export * from './Users';
export * from './User';
export * from './Logout';
export * from './Circle';
export * from './CheckLine';
export * from './Download';
export * from './Trash';
export * from './NewBadge';
export * from './Meal';
export * from './Dislike';
export * from './Profile';
export * from './Parking';
export * from './Wifi';
export * from './Pet';
export * from './Delivery';
export * from './Shipping';
export * from './Booking';
export * from './Star';
