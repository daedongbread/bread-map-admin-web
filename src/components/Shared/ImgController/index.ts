export * from './ImgManager';
export * from './ImgSelector';
