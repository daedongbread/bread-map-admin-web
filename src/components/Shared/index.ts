export * from './Header';
export * from './Input';
export * from './Button';
export * from './Pagination';
export * from './Preview';
export * from './SearchBar';
export * from './SelectBox';
export * from './Table';
export * from './Loading';
export * from './Error';
export * from './Tab';
export * from './ImgController';
