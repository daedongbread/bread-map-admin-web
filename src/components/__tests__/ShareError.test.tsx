import { Route } from 'react-router-dom';
import * as renderer from 'react-test-renderer';
import { Error } from '@/components/Shared';
import { withAllContexts, withRouter } from '@/tests';

describe('Shared Error', () => {
  it('에러 컴포넌트가 렌더링된다', () => {
    const component = renderTable();
    expect(component.toJSON()).toMatchSnapshot();
  });

  function renderTable() {
    return renderer.create(withAllContexts(withRouter(<Route path="/" element={<Error errMsg={'test'} />} />)));
  }
});
