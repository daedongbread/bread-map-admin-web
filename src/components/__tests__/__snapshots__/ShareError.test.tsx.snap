// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shared Error 에러 컴포넌트가 렌더링된다 1`] = `
<div
  className="css-13baf4u"
>
  <div
    className="css-qtzfs0"
  />
  <div
    className="css-1tfh94b"
  >
    <h2>
      test
    </h2>
  </div>
</div>
`;
